README.md

```diff
@@ -25,7 +25,7 @@ When deployed to an actual real-world production environment, they are often loc
 
 ## Setup
 
-- Install [.Net SDK 5.0](https://dotnet.microsoft.com/download/dotnet/5.0) (Linux / Windows / macOS)
+- Install .Net SDK [5.0](https://dotnet.microsoft.com/download/dotnet/5.0) and [6.0](https://dotnet.microsoft.com/download/dotnet/6.0) (Linux / Windows / macOS)
```

Comment:
and -> or

Reply:
Unfortunately, I don't have a choice until Bolero is migrated to .Net 6 (in a few days I hope).
.Net 5.0 is needed to run Wasm and I didn't find a way to target F# 6.0 without .Net 6.0 :s

Reply:
Ha, too clever by half as usual, sorry!